as-vending/functions/output.go

```diff
@@ -295,41 +300,47 @@ func (vendingState *VendingState) VerifyDoorAccess(edgexcontext *appcontext.Cont
 					vendingState.DoorClosedDuringCVWorkflow = false
 					vendingState.DoorOpenedDuringCVWorkflow = false
 					vendingState.InferenceDataReceived = false
-					edgexcontext.LoggingClient.Info("Maintenance Scan: ", "workflow:", vendingState.CVWorkflowStarted, "maintenance mode:", vendingState.MaintenanceMode, "open:", vendingState.DoorOpenedDuringCVWorkflow, "closed:", vendingState.DoorClosedDuringCVWorkflow, "Inference:", vendingState.InferenceDataReceived, "door:", vendingState.DoorClosed)
+					lc.Infof("Maintenance Scan: %v workflow: %v maintenance mode: %v open: %v closed: %v Inference: %v door: %v",
+						vendingState.CVWorkflowStarted,
+						vendingState.MaintenanceMode,
+						vendingState.DoorOpenedDuringCVWorkflow,
+						vendingState.DoorClosedDuringCVWorkflow,
+						vendingState.InferenceDataReceived,
+						vendingState.DoorClosed)
```

Comment:
Would this be better as separate logs statements. I.e reading this log message will be messy .
Also would use `+v` so the field labels are in the message also.